src/integrationTest/java/uk/gov/hmcts/reform/draftstore/controllers/helpers/SampleData.java

```diff
@@ -26,7 +26,7 @@ public static UpdateDraft updateDraft() {
         return new UpdateDraft("{}", null, "some_type");
     }
 
-    public static CreateDraft createDraft(int maxStaleDays) {
+    public static CreateDraft createDraft(Integer maxStaleDays) {
```

Comment:
I'd probably make it an Optional, but not that important.

Reply:
constructor below takes `Integer`